gnocchi/incoming/file.py

```diff
@@ -49,16 +49,14 @@ def set_storage_settings(self, num_sacks):
         data = {self.CFG_SACKS: num_sacks}
         with open(os.path.join(self.basepath_tmp, self.CFG_PREFIX), 'w') as f:
             json.dump(data, f)
-        utils.ensure_paths([self._sack_path(i)
-                            for i in six.moves.range(self.NUM_SACKS)])
+        utils.ensure_paths(map(self._sack_path, self.iter_sacks()))
```

Comment:
six.moves.map or list comprehension?

Reply:
really no difference here as it'll iterate on all so…